Tools/ApplicationDebugger/jacobi/README.md

```diff
@@ -0,0 +1,340 @@
+program `cpu`, `gpu`, or `accelerator` as the command-line argument.
+The selected device is displayed in the output.
+For the overview of the debugger, please refer to 
+[Get Started Guide](https://software.intel.com/en-us/get-started-with-debugging-dpcpp)
+of the application debugger.
+
+## Key Implementation Details
+
+The basic DPC++ implementation explained in the code includes device
+selection, buffer, accessor, and command groups.
+
+Each iteration of the Jacobi method performs the following computation:
+
+```
+x^{k+1} = D^{-1}(b - (A - D)x^k),
+```
+
+where `n x n` matrix `D` is a diagonal component of the matrix `A`.
+In the sample, this computation is offloaded to the device at each iteration.
+In the code, `x^{k+1}` corresponds to the variable `x_k1`, and
+`x^k` corresponds to `x_k`. At the end of each iteration we update `x_k` with
+the value of `x_k1`.
+
+## License
+
+Code samples are licensed under the MIT license. See
+[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+
+## Building and Running the `jacobi` Program
+
+> Note: if you have not already done so, set up your CLI 
+> environment by sourcing  the setvars script located in 
+> the root of your oneAPI installation. 
+>
+> Linux Sudo: . /opt/intel/oneapi/setvars.sh  
+> Linux User: . ~/intel/oneapi/setvars.sh  
+> Windows: C:\Program Files(x86)\Intel\oneAPI\setvars.bat
+
+### Setup
+
+Preliminary setup steps are needed for the debugger to function.
+Please see the setup instructions in the Get Started Guide based on
+your OS:
+[Linux](https://software.intel.com/en-us/get-started-with-debugging-dpcpp-linux),
+[Windows](https://software.intel.com/en-us/get-started-with-debugging-dpcpp-windows).
+
+
+### Include Files
+
+The include folder is located at
+`%ONEAPI_ROOT%\dev-utilities\latest\include` on your development
+system.
+
+
+### Running Samples In DevCloud
+
+If running a sample in the Intel DevCloud, remember that you must
+specify the compute node (CPU, GPU, FPGA) and whether to run in
+batch or interactive mode.  For the jacobi sample, a node
+with GPU and an interactive shell is recommended.
+
+```
+$ qsub -I -l nodes=1:gpu:ppn=2
+```
+
+For more information, see the Intel® oneAPI
+Base Toolkit Get Started Guide
+(https://devcloud.intel.com/oneapi/get-started/base-toolkit/).
+
+### On a Linux* System
+
+Perform the following steps:
+
+1.  Build the program using the following `cmake` commands.
+    ```
+    $ cd jacobi
+    $ mkdir build
+    $ cd build
+    $ cmake ..
+    $ make
+    ```
+    > Note: The cmake configuration enforces the `Debug` build type.
+
+2.  Run the bugged program:
+    ```
+    $ ./jacobi-bugged <device>
+    ```
+    > Note: `<device>` is the type of the device type to offload the kernel.
```

Comment:
Sentence needs restructuring

Reply:
I removed the second "type": `<device>` is the type of the device to offload the kernel. Ok with this?